StackExchange/Stacks
# docs(fundamentals): fix example-row-detail positioning

[STACKS-657](https://stackoverflow.atlassian.net/browse/STACKS-657)

---

This PR resolves positioning issues with the annotations used in color fundamentals docs page.


### Before

![image](https://github.com/user-attachments/assets/46837a09-f240-499f-b330-4753c7d1ebec)
![image](https://github.com/user-attachments/assets/80a5f8c7-e156-427c-9c97-4d94a2e7fa5e)


### After

![image](https://github.com/user-attachments/assets/d2cf37c9-b1c5-4f98-9670-74f68a3698d4)
![image](https://github.com/user-attachments/assets/0ccc7764-c139-45d9-ac71-5783a7584435)

### How to test

1. Spin this PR up locally (`npm run start`)
2. Head to the [_Color Fundamentals page_](http://localhost:8080/product/foundation/color-fundamentals)
3. Verify that the annotations under each of the [monochromatic color strips](http://localhost:8080/product/foundation/color-fundamentals/#backgrounds) are positioned as expected
4. **Bonus**: Since this fix necessitated changes to the [accessibility docs](http://localhost:8080/product/foundation/accessibility/#foundation-for-custom-approach) code, verify that annotations under examples are positioned as expected
